### PR TITLE
Feature(UI): Styling

### PR DESCRIPTION
**Pull request by blessedjasonmwanza**
## Feature(UI): Styling
In this UI update:

- Style Project UI following [Zipline Design](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek.md)
- Apply React best practices and language style guides in code.
- Add Project preview Image in ReadMe

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ export default function AddBook() {
       category: form.book_category.value,
       author: '',
     };
+    e.target.reset();
     dispatch(addBook(newBook));
   };
   return (
```
Comment (Emmyn5600) [inline src/components/AddBook.js:20]: - [ ] Let us try to match with the design provided. Kindly, compare your implementation with the design provided to meet all the requirements. Please use the same font size, color, and font family.

Check the header section and the add book button

Original design

db8png

Yours 
db5

Original design

db9

Your design

db6

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @blessedjasonmwanza 

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.

 ## Required Changes ♻️
- Check the comments under the review.

## Optional suggestions

- n/a

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/components/AddBook.js (hunk 1 above)
Comment (blessedjasonmwanza) [reply src/components/AddBook.js:20]: Hello @Emmyn5600 ,

Thanks a lot for your insightful review.

On to make the requested changes right away. 🏃🏽

**Comment by omar-labana** [comment]
Hi @blessedjasonmwanza, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**